### ISSUE TITLE
[@angular-eslint/use-component-selector] Component Selector does not support Backticks 

### ISSUE THREAD

**Issue by Raju** [angular-eslint/angular-eslint#800]
**Description and reproduction of the issue**

When the selector name is in doublequotes ("app-root") it does not throw any error.

But when selector name is in Backticks (`app-root`) it throws an error -> "The selector of the component is mandatoryeslint@angular-eslint/use-component-selector"

![image](https://user-images.githubusercontent.com/42042961/141680278-aa371893-17f0-4612-8dc5-734e92e6e081.png)

```JSON
{
  "rules": {
    "@angular-eslint/<rule>": "@angular-eslint/use-component-selector"
  }
}
```

```TS
@Component ( {
  selector    : `app-root`,
  templateUrl : `./app.component.html`,
  styleUrls   : [ `./app.component.scss` ],
} )
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/eslint-plugin`    |"13.0.0-alpha.0"|
| `@typescript-eslint/parser`       |"^5.3.1"|
| `ESLint`                           |"^8.2.0"|
| `node`                           |"v16.13.0"|

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 13.0.2
Node: 16.13.0
Package Manager: npm 8.1.3
OS: linux x64

Angular: 13.0.1
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1300.2
@angular-devkit/build-angular   13.0.2
@angular-devkit/core            13.0.2
@angular-devkit/schematics      13.0.2
@angular/cli                    13.0.2
@schematics/angular             13.0.2
rxjs                            7.4.0
typescript                      4.4.4

```

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.
